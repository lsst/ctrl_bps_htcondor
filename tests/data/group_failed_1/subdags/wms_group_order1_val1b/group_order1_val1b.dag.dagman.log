000 (10113.000.000) 2025-03-10 11:07:01 Job submitted from host: <10.0.0.33:20939?addrs=10.0.0.33-20939+[2601-248-8500-b50--617a]-20939&alias=test02>
    DAG Node: wms_group_order1_val1b
...
001 (10113.000.000) 2025-03-10 11:07:01 Job executing on host: <10.0.0.33:20939?addrs=10.0.0.33-20939+[2601-248-8500-b50--617a]-20939&alias=test02>
...
005 (10113.000.000) 2025-03-10 11:07:34 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	0  -  Run Bytes Sent By Job
	0  -  Run Bytes Received By Job
	0  -  Total Bytes Sent By Job
	0  -  Total Bytes Received By Job
...
