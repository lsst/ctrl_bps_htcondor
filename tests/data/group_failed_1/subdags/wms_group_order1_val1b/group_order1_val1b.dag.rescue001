# Rescue DAG file, created after running
#   the group_order1_val1b.dag DAG file
# Created 3/10/2025 16:07:34 UTC
# Rescue DAG version: 2.0.1 (partial)
#
# Total number of Nodes: 4
# Nodes premarked DONE: 2
# Nodes that failed: 1
#   label2_val1b_val2b,<ENDLIST>

DONE label2_val1b_val2a
DONE label4_val1b_val2a
