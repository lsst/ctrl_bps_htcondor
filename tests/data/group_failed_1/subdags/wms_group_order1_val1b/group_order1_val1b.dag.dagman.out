03/10/25 11:07:01 ******************************************************
03/10/25 11:07:01 ** condor_scheduniv_exec.10113.0 (CONDOR_DAGMAN) STARTING UP
03/10/25 11:07:01 ** /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_dagman
03/10/25 11:07:01 ** SubsystemInfo: name=DAGMAN type=DAGMAN(9) class=CLIENT(2)
03/10/25 11:07:01 ** Configuration: subsystem:DAGMAN local:<NONE> class:CLIENT
03/10/25 11:07:01 ** $CondorVersion: 24.0.4 2025-02-10 $
03/10/25 11:07:01 ** $CondorPlatform: X86_64-AlmaLinux_9.5 $
03/10/25 11:07:01 ** PID = 18511 RealUID = 27031
03/10/25 11:07:01 ** Log last touched time unavailable (No such file or directory)
03/10/25 11:07:01 ******************************************************
03/10/25 11:07:01 Using config source: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/etc/condor/condor_config
03/10/25 11:07:01 Using local config sources: 
03/10/25 11:07:01    /etc/condor/condor_config
03/10/25 11:07:01    /etc/condor/config.d/00-minicondor
03/10/25 11:07:01    /etc/condor/config.d/00-security
03/10/25 11:07:01    /etc/condor/config.d/10-stash-plugin.conf
03/10/25 11:07:01    /etc/condor/config.d/99-lsst
03/10/25 11:07:01    /etc/condor/condor_config.local
03/10/25 11:07:01    /home/testuser/.condor/user_config
03/10/25 11:07:01 config Macros = 99, Sorted = 99, StringBytes = 3220, TablesBytes = 3652
03/10/25 11:07:01 CLASSAD_CACHING is ENABLED
03/10/25 11:07:01 Daemon Log is logging: D_ALWAYS D_ERROR D_STATUS
03/10/25 11:07:01 DaemonCore: No command port requested.
03/10/25 11:07:01 DAGMAN_USE_STRICT setting: 1
03/10/25 11:07:01 DAGMAN_VERBOSITY setting: 3
03/10/25 11:07:01 DAGMAN_DEBUG_CACHE_SIZE setting: 5242880
03/10/25 11:07:01 DAGMAN_DEBUG_CACHE_ENABLE setting: False
03/10/25 11:07:01 DAGMAN_SUBMIT_DELAY setting: 0
03/10/25 11:07:01 DAGMAN_MAX_SUBMIT_ATTEMPTS setting: 6
03/10/25 11:07:01 DAGMAN_STARTUP_CYCLE_DETECT setting: False
03/10/25 11:07:01 DAGMAN_MAX_SUBMITS_PER_INTERVAL setting: 100
03/10/25 11:07:01 DAGMAN_AGGRESSIVE_SUBMIT setting: False
03/10/25 11:07:01 DAGMAN_USER_LOG_SCAN_INTERVAL setting: 1
03/10/25 11:07:01 DAGMAN_QUEUE_UPDATE_INTERVAL setting: 30
03/10/25 11:07:01 DAGMAN_DEFAULT_PRIORITY setting: 0
03/10/25 11:07:01 DAGMAN_SUPPRESS_NOTIFICATION setting: False
03/10/25 11:07:01 allow_events (DAGMAN_ALLOW_EVENTS) setting: 114
03/10/25 11:07:01 DAGMAN_RETRY_SUBMIT_FIRST setting: True
03/10/25 11:07:01 DAGMAN_RETRY_NODE_FIRST setting: False
03/10/25 11:07:01 DAGMAN_MAX_JOBS_IDLE setting: 1000
03/10/25 11:07:01 DAGMAN_MAX_JOBS_SUBMITTED setting: 0
03/10/25 11:07:01 DAGMAN_MAX_PRE_SCRIPTS setting: 20
03/10/25 11:07:01 DAGMAN_MAX_POST_SCRIPTS setting: 20
03/10/25 11:07:01 DAGMAN_MAX_HOLD_SCRIPTS setting: 20
03/10/25 11:07:01 DAGMAN_MUNGE_NODE_NAMES setting: True
03/10/25 11:07:01 DAGMAN_PROHIBIT_MULTI_JOBS setting: False
03/10/25 11:07:01 DAGMAN_SUBMIT_DEPTH_FIRST setting: False
03/10/25 11:07:01 DAGMAN_ALWAYS_RUN_POST setting: False
03/10/25 11:07:01 DAGMAN_CONDOR_SUBMIT_EXE setting: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_submit
03/10/25 11:07:01 DAGMAN_CONDOR_RM_EXE setting: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_rm
03/10/25 11:07:01 DAGMAN_USE_DIRECT_SUBMIT setting: True
03/10/25 11:07:01 DAGMAN_PRODUCE_JOB_CREDENTIALS setting: True
03/10/25 11:07:01 DAGMAN_DEFAULT_APPEND_VARS setting: False
03/10/25 11:07:01 DAGMAN_INHERIT_ATTRS_PREFIX setting: 
03/10/25 11:07:01 DAGMAN_INHERIT_ATTRS setting: 
03/10/25 11:07:01 DAGMAN_ABORT_DUPLICATES setting: True
03/10/25 11:07:01 DAGMAN_ABORT_ON_SCARY_SUBMIT setting: True
03/10/25 11:07:01 DAGMAN_PENDING_REPORT_INTERVAL setting: 60
03/10/25 11:07:01 DAGMAN_CHECK_QUEUE_INTERVAL setting: 28800
03/10/25 11:07:01 DAGMAN_AUTO_RESCUE setting: True
03/10/25 11:07:01 DAGMAN_MAX_RESCUE_NUM setting: 100
03/10/25 11:07:01 DAGMAN_WRITE_PARTIAL_RESCUE setting: True
03/10/25 11:07:01 DAGMAN_DEFAULT_NODE_LOG setting: @(DAG_DIR)/@(DAG_FILE).nodes.log
03/10/25 11:07:01 DAGMAN_GENERATE_SUBDAG_SUBMITS setting: True
03/10/25 11:07:01 DAGMAN_MAX_JOB_HOLDS setting: 100
03/10/25 11:07:01 DAGMAN_HOLD_CLAIM_TIME setting: 20
03/10/25 11:07:01 ALL_DEBUG setting: D_FULLDEBUG
03/10/25 11:07:01 DAGMAN_SUPPRESS_JOB_LOGS setting: False
03/10/25 11:07:01 DAGMAN_REMOVE_NODE_JOBS setting: True
03/10/25 11:07:01 DAGMAN_REMOVE_JOBS_AFTER_LIMIT_CHANGE setting: False
03/10/25 11:07:01 DAGMAN will adjust edges after parsing
03/10/25 11:07:01 argv[0] == "condor_scheduniv_exec.10113.0"
03/10/25 11:07:01 argv[1] == "-Lockfile"
03/10/25 11:07:01 argv[2] == "group_order1_val1b.dag.lock"
03/10/25 11:07:01 argv[3] == "-Dag"
03/10/25 11:07:01 argv[4] == "group_order1_val1b.dag"
03/10/25 11:07:01 argv[5] == "-MaxIdle"
03/10/25 11:07:01 argv[6] == "1000"
03/10/25 11:07:01 argv[7] == "-CsdVersion"
03/10/25 11:07:01 argv[8] == "$CondorVersion: 24.0.4 2025-02-10 $"
03/10/25 11:07:01 argv[9] == "-dagman"
03/10/25 11:07:01 argv[10] == "/work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_dagman"
03/10/25 11:07:01 argv[11] == "-AutoRescue"
03/10/25 11:07:01 argv[12] == "1"
03/10/25 11:07:01 argv[13] == "-DoRescueFrom"
03/10/25 11:07:01 argv[14] == "0"
03/10/25 11:07:01 argv[15] == "-do_recurse"
03/10/25 11:07:01 argv[16] == "-update_submit"
03/10/25 11:07:01 Workflow batch-id: <10093.0>
03/10/25 11:07:01 Workflow batch-name: <u_testuser_DM-46294_group_fail_20250310T160455Z.dag+10093>
03/10/25 11:07:01 Workflow accounting_group: <>
03/10/25 11:07:01 Workflow accounting_group_user: <>
03/10/25 11:07:01 DAGMAN_LOG_ON_NFS_IS_ERROR setting: False
03/10/25 11:07:01 Default node log file is: </work/testuser/gen3work/ordering-46294/subdag/submit/u/testuser/DM-46294/group/fail/20250310T160455Z/subdags/wms_group_order1_val1b/./group_order1_val1b.dag.nodes.log>
03/10/25 11:07:01 DAG Lockfile will be written to group_order1_val1b.dag.lock
03/10/25 11:07:01 DAG Input file is group_order1_val1b.dag
03/10/25 11:07:01 Parsing 1 dagfiles
03/10/25 11:07:01 Parsing group_order1_val1b.dag ...
03/10/25 11:07:01 Adjusting edges
03/10/25 11:07:01 Dag contains 4 total nodes
03/10/25 11:07:01 Bootstrapping...
03/10/25 11:07:01 Number of pre-completed nodes: 0
03/10/25 11:07:01 MultiLogFiles: truncating log file /work/testuser/gen3work/ordering-46294/subdag/submit/u/testuser/DM-46294/group/fail/20250310T160455Z/subdags/wms_group_order1_val1b/./group_order1_val1b.dag.nodes.log
03/10/25 11:07:01 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:01 Of 4 nodes total:
03/10/25 11:07:01  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:01   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:01     0       0        0       0       2          2        0        0
03/10/25 11:07:01 0 job proc(s) currently held
03/10/25 11:07:01 DAGMan Runtime Statistics: [EventCycleTimeCount = 0.0; EventCycleTimeSum = 0.0; LogProcessCycleTimeCount = 0.0; LogProcessCycleTimeSum = 0.0; SleepCycleTimeCount = 0.0; SleepCycleTimeSum = 0.0; SubmitCycleTimeCount = 0.0; SubmitCycleTimeSum = 0.0; ]
03/10/25 11:07:01 Registering condor_event_timer...
03/10/25 11:07:02 Submitting HTCondor Node label2_val1b_val2b job(s)...
03/10/25 11:07:02 Submitting node label2_val1b_val2b from file label2_val1b_val2b.sub using direct job submission
03/10/25 11:07:02 	assigned HTCondor ID (10115.0.0)
03/10/25 11:07:02 Submitting HTCondor Node label2_val1b_val2a job(s)...
03/10/25 11:07:02 Submitting node label2_val1b_val2a from file label2_val1b_val2a.sub using direct job submission
03/10/25 11:07:03 	assigned HTCondor ID (10116.0.0)
03/10/25 11:07:03 Just submitted 2 jobs this cycle...
03/10/25 11:07:03 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:03 Of 4 nodes total:
03/10/25 11:07:03  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:03   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:03     0       0        2       0       0          2        0        0
03/10/25 11:07:03 0 job proc(s) currently held
03/10/25 11:07:03 DAGMan Runtime Statistics: [EventCycleTimeCount = 0.0; EventCycleTimeSum = 0.0; LogProcessCycleTimeCount = 0.0; LogProcessCycleTimeSum = 0.0; SleepCycleTimeCount = 0.0; SleepCycleTimeSum = 0.0; SubmitCycleTimeAvg = 0.1422841548919678; SubmitCycleTimeCount = 1.0; SubmitCycleTimeMax = 0.1422841548919678; SubmitCycleTimeMin = 0.1422841548919678; SubmitCycleTimeStd = 0.1422841548919678; SubmitCycleTimeSum = 0.1422841548919678; ]
03/10/25 11:07:04 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:04 Reassigning the id of job label2_val1b_val2b from (10115.0.0) to (10115.0.0)
03/10/25 11:07:04 Event: ULOG_SUBMIT for HTCondor Node label2_val1b_val2b (10115.0.0) {03/10/25 11:07:02}
03/10/25 11:07:04 Number of idle job procs: 1
03/10/25 11:07:04 Reassigning the id of job label2_val1b_val2a from (10116.0.0) to (10116.0.0)
03/10/25 11:07:04 Event: ULOG_SUBMIT for HTCondor Node label2_val1b_val2a (10116.0.0) {03/10/25 11:07:02}
03/10/25 11:07:04 Number of idle job procs: 2
03/10/25 11:07:08 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:08 Event: ULOG_EXECUTE for HTCondor Node label2_val1b_val2b (10115.0.0) {03/10/25 11:07:07}
03/10/25 11:07:08 Number of idle job procs: 1
03/10/25 11:07:11 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:11 Event: ULOG_EXECUTE for HTCondor Node label2_val1b_val2a (10116.0.0) {03/10/25 11:07:10}
03/10/25 11:07:11 Number of idle job procs: 0
03/10/25 11:07:19 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:19 Event: ULOG_JOB_TERMINATED for HTCondor Node label2_val1b_val2b (10115.0.0) {03/10/25 11:07:18}
03/10/25 11:07:19 Number of idle job procs: 0
03/10/25 11:07:19 Node label2_val1b_val2b job proc (10115.0.0) failed with status 1.
03/10/25 11:07:19 DAG status: 2 (DAG_STATUS_NODE_FAILED)
03/10/25 11:07:19 Of 4 nodes total:
03/10/25 11:07:19  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:19   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:19     0       0        1       0       0          1        1        1
03/10/25 11:07:19 0 job proc(s) currently held
03/10/25 11:07:19 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.008977130055427551; EventCycleTimeCount = 16.0; EventCycleTimeMax = 0.1426432132720947; EventCycleTimeMin = 1.811981201171875E-05; EventCycleTimeStd = 0.03564436837611067; EventCycleTimeSum = 0.1436340808868408; LogProcessCycleTimeAvg = 0.0001839399337768555; LogProcessCycleTimeCount = 4.0; LogProcessCycleTimeMax = 0.0002329349517822266; LogProcessCycleTimeMin = 0.0001358985900878906; LogProcessCycleTimeStd = 3.967144738976877E-05; LogProcessCycleTimeSum = 0.0007357597351074219; SleepCycleTimeAvg = 1.001089125871658; SleepCycleTimeCount = 16.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.001067876815796; SleepCycleTimeStd = 1.636551508620439E-05; SleepCycleTimeSum = 16.01742601394653; SubmitCycleTimeAvg = 0.008372334872975069; SubmitCycleTimeCount = 17.0; SubmitCycleTimeMax = 0.1422841548919678; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.03450828616730397; SubmitCycleTimeSum = 0.1423296928405762; ]
03/10/25 11:07:23 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:23 Event: ULOG_JOB_TERMINATED for HTCondor Node label2_val1b_val2a (10116.0.0) {03/10/25 11:07:22}
03/10/25 11:07:23 Number of idle job procs: 0
03/10/25 11:07:23 Node label2_val1b_val2a job proc (10116.0.0) completed successfully.
03/10/25 11:07:23 Node label2_val1b_val2a job completed
03/10/25 11:07:23 DAG status: 2 (DAG_STATUS_NODE_FAILED)
03/10/25 11:07:23 Of 4 nodes total:
03/10/25 11:07:23  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:23   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:23     1       0        0       0       1          0        1        1
03/10/25 11:07:23 0 job proc(s) currently held
03/10/25 11:07:23 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.007202458381652832; EventCycleTimeCount = 20.0; EventCycleTimeMax = 0.1426432132720947; EventCycleTimeMin = 1.811981201171875E-05; EventCycleTimeStd = 0.03187958539909194; EventCycleTimeSum = 0.1440491676330566; LogProcessCycleTimeAvg = 0.0001853466033935547; LogProcessCycleTimeCount = 5.0; LogProcessCycleTimeMax = 0.0002329349517822266; LogProcessCycleTimeMin = 0.0001358985900878906; LogProcessCycleTimeStd = 3.45001652249974E-05; LogProcessCycleTimeSum = 0.0009267330169677734; SleepCycleTimeAvg = 1.001087892055512; SleepCycleTimeCount = 20.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.001060962677002; SleepCycleTimeStd = 1.738012464282954E-05; SleepCycleTimeSum = 20.02175784111023; SubmitCycleTimeAvg = 0.006778138024466378; SubmitCycleTimeCount = 21.0; SubmitCycleTimeMax = 0.1422841548919678; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.03104832898541014; SubmitCycleTimeSum = 0.1423408985137939; ]
03/10/25 11:07:24 Submitting HTCondor Node label4_val1b_val2a job(s)...
03/10/25 11:07:24 Submitting node label4_val1b_val2a from file label4_val1b_val2a.sub using direct job submission
03/10/25 11:07:24 	assigned HTCondor ID (10119.0.0)
03/10/25 11:07:24 Just submitted 1 job this cycle...
03/10/25 11:07:24 DAG status: 2 (DAG_STATUS_NODE_FAILED)
03/10/25 11:07:24 Of 4 nodes total:
03/10/25 11:07:24  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:24   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:24     1       0        1       0       0          0        1        1
03/10/25 11:07:24 0 job proc(s) currently held
03/10/25 11:07:24 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.006876059940883091; EventCycleTimeCount = 21.0; EventCycleTimeMax = 0.1426432132720947; EventCycleTimeMin = 1.811981201171875E-05; EventCycleTimeStd = 0.03110835614014297; EventCycleTimeSum = 0.1443972587585449; LogProcessCycleTimeAvg = 0.0001853466033935547; LogProcessCycleTimeCount = 5.0; LogProcessCycleTimeMax = 0.0002329349517822266; LogProcessCycleTimeMin = 0.0001358985900878906; LogProcessCycleTimeStd = 3.45001652249974E-05; LogProcessCycleTimeSum = 0.0009267330169677734; SleepCycleTimeAvg = 1.001088040215629; SleepCycleTimeCount = 21.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.001060962677002; SleepCycleTimeStd = 1.695365036120837E-05; SleepCycleTimeSum = 21.0228488445282; SubmitCycleTimeAvg = 0.009547504511746492; SubmitCycleTimeCount = 22.0; SubmitCycleTimeMax = 0.1422841548919678; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.03296696266755786; SubmitCycleTimeSum = 0.2100450992584229; ]
03/10/25 11:07:25 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:25 Reassigning the id of job label4_val1b_val2a from (10119.0.0) to (10119.0.0)
03/10/25 11:07:25 Event: ULOG_SUBMIT for HTCondor Node label4_val1b_val2a (10119.0.0) {03/10/25 11:07:24}
03/10/25 11:07:25 Number of idle job procs: 1
03/10/25 11:07:25 Event: ULOG_EXECUTE for HTCondor Node label4_val1b_val2a (10119.0.0) {03/10/25 11:07:24}
03/10/25 11:07:25 Number of idle job procs: 0
03/10/25 11:07:34 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:34 Event: ULOG_JOB_TERMINATED for HTCondor Node label4_val1b_val2a (10119.0.0) {03/10/25 11:07:33}
03/10/25 11:07:34 Number of idle job procs: 0
03/10/25 11:07:34 Node label4_val1b_val2a job proc (10119.0.0) completed successfully.
03/10/25 11:07:34 Node label4_val1b_val2a job completed
03/10/25 11:07:34 DAG status: 2 (DAG_STATUS_NODE_FAILED)
03/10/25 11:07:34 Of 4 nodes total:
03/10/25 11:07:34  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:34   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:34     2       0        0       0       0          0        1        1
03/10/25 11:07:34 0 job proc(s) currently held
03/10/25 11:07:34 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.007200210325179562; EventCycleTimeCount = 31.0; EventCycleTimeMax = 0.1426432132720947; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.02796516262792155; EventCycleTimeSum = 0.2232065200805664; LogProcessCycleTimeAvg = 0.0001920972551618303; LogProcessCycleTimeCount = 7.0; LogProcessCycleTimeMax = 0.0002470016479492188; LogProcessCycleTimeMin = 0.0001358985900878906; LogProcessCycleTimeStd = 3.752946600838023E-05; LogProcessCycleTimeSum = 0.001344680786132812; SleepCycleTimeAvg = 1.00108447382527; SleepCycleTimeCount = 31.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.001057147979736; SleepCycleTimeStd = 1.812019434011055E-05; SleepCycleTimeSum = 31.03361868858337; SubmitCycleTimeAvg = 0.006564684212207794; SubmitCycleTimeCount = 32.0; SubmitCycleTimeMax = 0.1422841548919678; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.02750342926485443; SubmitCycleTimeSum = 0.2100698947906494; ]
03/10/25 11:07:34 ERROR: the following job(s) failed:
03/10/25 11:07:34 ---------------------- Node ----------------------
03/10/25 11:07:34       Node Name: label2_val1b_val2b
03/10/25 11:07:34            Noop: false
03/10/25 11:07:34          NodeID: 0
03/10/25 11:07:34     Node Status: STATUS_ERROR    
03/10/25 11:07:34 Node return val: 1
03/10/25 11:07:34           Error: Job proc (10115.0.0) failed with status 1
03/10/25 11:07:34 Job Submit File: label2_val1b_val2b.sub
03/10/25 11:07:34  HTCondor Job ID: (10115.0.0)
03/10/25 11:07:34 PARENTS:  WAITING: 0 CHILDREN: label4_val1b_val2b
03/10/25 11:07:34 ---------------------------------------	<END>
03/10/25 11:07:34 Aborting DAG...
03/10/25 11:07:34 Writing Rescue DAG to group_order1_val1b.dag.rescue001...
03/10/25 11:07:34 Removing submitted jobs...
03/10/25 11:07:34 Removing any/all submitted HTCondor jobs...
03/10/25 11:07:34 Running: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_rm -const DAGManJobId==10113 -reason DAG' 'Abort:' 'DAG' 'is' 'exiting' 'and' 'writing' 'rescue' 'file.
03/10/25 11:07:34 Note: 0 total node deferrals because of -MaxJobs limit (0)
03/10/25 11:07:34 Note: 0 total node deferrals because of -MaxIdle limit (1000)
03/10/25 11:07:34 Note: 0 total node deferrals because of node category throttles
03/10/25 11:07:34 Note: 0 total PRE script deferrals because of -MaxPre limit (20) or DEFER
03/10/25 11:07:34 Note: 0 total POST script deferrals because of -MaxPost limit (20) or DEFER
03/10/25 11:07:34 Note: 0 total HOLD script deferrals because of -MaxHold limit (20) or DEFER
03/10/25 11:07:34 DAG status: 2 (DAG_STATUS_NODE_FAILED)
03/10/25 11:07:34 Of 4 nodes total:
03/10/25 11:07:34  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:34   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:34     2       0        0       0       0          0        1        1
03/10/25 11:07:34 0 job proc(s) currently held
03/10/25 11:07:34 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.007200210325179562; EventCycleTimeCount = 31.0; EventCycleTimeMax = 0.1426432132720947; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.02796516262792155; EventCycleTimeSum = 0.2232065200805664; LogProcessCycleTimeAvg = 0.0001920972551618303; LogProcessCycleTimeCount = 7.0; LogProcessCycleTimeMax = 0.0002470016479492188; LogProcessCycleTimeMin = 0.0001358985900878906; LogProcessCycleTimeStd = 3.752946600838023E-05; LogProcessCycleTimeSum = 0.001344680786132812; SleepCycleTimeAvg = 1.00108447382527; SleepCycleTimeCount = 31.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.001057147979736; SleepCycleTimeStd = 1.812019434011055E-05; SleepCycleTimeSum = 31.03361868858337; SubmitCycleTimeAvg = 0.006564684212207794; SubmitCycleTimeCount = 32.0; SubmitCycleTimeMax = 0.1422841548919678; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.02750342926485443; SubmitCycleTimeSum = 0.2100698947906494; ]
03/10/25 11:07:34 Wrote metrics file group_order1_val1b.dag.metrics.
03/10/25 11:07:34 DAGMan Runtime Statistics: [EventCycleTimeAvg = 0.007200210325179562; EventCycleTimeCount = 31.0; EventCycleTimeMax = 0.1426432132720947; EventCycleTimeMin = 1.406669616699219E-05; EventCycleTimeStd = 0.02796516262792155; EventCycleTimeSum = 0.2232065200805664; LogProcessCycleTimeAvg = 0.0001920972551618303; LogProcessCycleTimeCount = 7.0; LogProcessCycleTimeMax = 0.0002470016479492188; LogProcessCycleTimeMin = 0.0001358985900878906; LogProcessCycleTimeStd = 3.752946600838023E-05; LogProcessCycleTimeSum = 0.001344680786132812; SleepCycleTimeAvg = 1.00108447382527; SleepCycleTimeCount = 31.0; SleepCycleTimeMax = 1.001116991043091; SleepCycleTimeMin = 1.001057147979736; SleepCycleTimeStd = 1.812019434011055E-05; SleepCycleTimeSum = 31.03361868858337; SubmitCycleTimeAvg = 0.006564684212207794; SubmitCycleTimeCount = 32.0; SubmitCycleTimeMax = 0.1422841548919678; SubmitCycleTimeMin = 1.192092895507812E-06; SubmitCycleTimeStd = 0.02750342926485443; SubmitCycleTimeSum = 0.2100698947906494; ]
03/10/25 11:07:34 **** condor_scheduniv_exec.10113.0 (condor_DAGMAN) pid 18511 EXITING WITH STATUS 1
